main.go

```diff
@@ -206,10 +206,19 @@ func overrideDefaults(defaults *Options, args []string) {
 func parseDefaultConfigs(parser *flags.Parser) error {
 	app := xdg.App{Name: "yab"}
 
-	configFile := app.ConfigPath("defaults.ini")
-	if _, err := os.Stat(configFile); err != nil {
-		// No defaults file to read
-		return nil
+	// Find the best config path to use, preferring the user's config path and
```

Comment:
Was just thinking about this. The way XDG works for all of the files on my Linux desktop is that it does not merge files, it uses the same precedence logic I've implemented here. The recommended way to customize a system XDG file on Linux is to copy the system one to your home directory and then modify it.

The issue here is that some of the flags we use with yab do something only if set. That means that you can't meaningfully unset a flag by putting it in your home directory and clearing that value.